.mage/js_sdk.go

```diff
@@ -58,8 +58,13 @@ func (k JsSDK) Deps() error {
 
 // Build builds the source files and output into 'dist'.
 func (k JsSDK) Build() error {
+	isCI := os.Getenv("CI") == "true"
```

Comment:
This should not be necessary

Reply:
See https://github.com/TheThingsNetwork/lorawan-stack/pull/2764#discussion_r442275560